src/Core/Domain/Country/ValueObject/CountryZipCodeFormat.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Country\ValueObject;
+
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\TypedRegex;
+use PrestaShop\PrestaShop\Core\Domain\Country\Exception\CountryConstraintException;
+
+/**
+ * Provides valid country zip code
+ */
+class CountryZipCodeFormat
+{
+    /**
+     * Zip code format regexp validation pattern
+     */
+    const ZIP_CODE_PATTERN = '/^[NLCnlc 0-9-]+$/';
```

Comment:
What is the scope of this constant ?

Reply:
It is used to validate if zip code format is correct. It used to be hard-coded but now i m adding const for it.

Reply:
I believe this is what @Progi1984 asked about
```suggestion
    public const ZIP_CODE_PATTERN = '/^[NLCnlc 0-9-]+$/';
```

Reply:
Indeed, it's better to always explicitly set the const visibility